content/12-target-details.md

```diff
@@ -1945,6 +1945,52 @@ java -jar bin/Main.jar
 
 
 
+<!--label:target-jvm-->
+### JVM
+
+<!--subtoc-->
+
+<!--label:target-jvm-getting-started-->
+#### Getting started with Haxe/JVM
+
+To get started with Haxe/JVM, create a new folder and save this class as `Main.hx`.
+
+[code asset](assets/HelloWorld.hx)
+
+To compile Haxe to JVM bytecode we need two obvious prerequisites installed:
+
+* [hxjava haxelib](http://lib.haxe.org/p/hxjava). This is the support library for the Java backend of the Haxe compiler.
+* [JRE - Java Runtime Environment](https://java.com/download/).
```

Comment:
I think this is not required to compile to jvm.

Reply:
It's required to run a compiled jar.

Reply:
Ok, clarified this.